docs/LANGUAGE.md

```diff
@@ -0,0 +1,388 @@
+# Opalog: OPA's Query Language
+
+OPA includes a policy engine which is purpose built for reasoning about information represented in structured documents such as JSON. Data stored in in the policy engine can be queried using OPA's native query language: Opalog.
```

Comment:
One of my few pet peeves in english: that vs. which.  Here's my understanding.  "that" is for part of a sentence that cannot be removed--it helps define the subject.  "which" is for part of a sentence that could be removed, and is preceded by a comma or used as a parenthetical.  Many places throughout the doc use "which" where "that" seems correct.